x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.tsx

```diff
@@ -377,19 +377,24 @@ const DetectionEnginePageComponent: React.FC<DetectionEngineComponentProps> = ({
     ]
   );
 
+  const allAlertTableFilters = useMemo(
+    () => alertsTableDefaultFilters,
+    [alertsTableDefaultFilters]
+  );
   const renderAlertTable = useCallback(
     (groupingFilters: Filter[]) => {
+      allAlertTableFilters.push(...groupingFilters);
```

Comment:
Probably not actually ideal as we may want a new ref here if the array contents change, but as of right now every time this is called on a render a new array will be created irrespective of the contents.